packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts

```diff
@@ -0,0 +1,26 @@
+import { loadSchema } from '../../../src/generator/utils/loading';
+
+describe('loading', () => {
+  it('JSON schema', () => {
+    const graphqlSchema = loadSchema(__dirname + '/../../../fixtures/schema.json');
```

Comment:
We should probably use Node's `path` module to create these paths.

Reply:
Thanks for your review.
I fixed it.